config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -769,6 +780,17 @@ protected void configure(ServerHttpSecurity http) {
 					.addFilterAt(oauth2, SecurityWebFiltersOrder.AUTHENTICATION);
 			}
 
+			protected ReactiveAuthenticationManager getAuthenticationManager() {
+				if (this.authenticationManager == null) {
```

Comment:
It is easier to read if there is less code indented. Instead of a relatively large if block either extract this into a create method or use `if (this.authenticationManager != null) return this.authenticationManager`